examples/metalk8s-solution-example/operator/pkg/config/config.go

```diff
@@ -0,0 +1,153 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"strings"
+
+	"gopkg.in/go-playground/validator.v9"
+	"gopkg.in/yaml.v2"
+)
+
+type Repository struct {
+	Endpoint string `yaml:"endpoint" validate:"required"`
+	Images   []string `yaml:"images" validate:"required,ne=0"`
+}
+
+type OperatorConfig struct {
+	Kind         string `yaml:"kind" validate:"required,eq=OperatorConfig"`
+	APIVersion   string `yaml:"apiVersion" validate:"required"`
```

Comment:
Would be nice to validate we know the APIVersion as well ;)

Reply:
Yes, sure.

Reply:
done